.github/instructions/dsc-community-style-guidelines.instructions.md

```
@@ -0,0 +1,34 @@
+---
+description: Core DSC community guidelines for AI development.
+applyTo: "**"
+---
+
+# DSC Community Guidelines
+
+## Terminology
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)